pkg/injector/injector.go

```diff
@@ -254,12 +255,14 @@ func (i *injector) handleRequest(w http.ResponseWriter, r *http.Request) {
 			http.StatusInternalServerError,
 		)
 
+		log.Errorf("Sidecar injector failed to inject for app '%s'", diagAppID)
```

Comment:
same.

Reply:
We do already at 250, but I guess it should be one log line. Will fix.